tests/system/python/packages/test_filters.py

```diff
@@ -0,0 +1,660 @@
+    res = res.read().decode()
+    r = json.loads(res)
+    return r
+
+
+def get_request(foglamp_url, get_url):
+    con = http.client.HTTPConnection(foglamp_url)
+    con.request("GET", get_url)
+    resp = con.getresponse()
+    r = json.loads(resp.read().decode())
+    return r
+
+
+def put_request(foglamp_url, put_url, payload):
+    conn = http.client.HTTPConnection(foglamp_url)
+    conn.request("PUT", put_url, json.dumps(payload))
+    res = conn.getresponse()
+    assert 200 == res.status
+    r = json.loads(res.read().decode())
+    return r
+
+
+# TODO : set PROJECT_ROOT appropriately
+def call_fogbench(wait_time):
+    execute_fogbench='cd $PROJECT_ROOT/extras/python ;python3 -m fogbench -t $FOGLAMP_ROOT/data/template.json ' \
+                     '-p http -O 10'
+    exit_code = os.system(execute_fogbench)
+    assert exit_code == 0
+    time.sleep(wait_time)
+
+
+def add_south_http(foglamp_url, name):
+    data = {"name": name, "type": "south", "plugin": "http_south", "enabled": True, "config": {}}
+    post_url = "/foglamp/service"
+    post_request(foglamp_url, post_url, data)
+
+
+def verify_south_added(foglamp_url, name, retries):
+    get_url = "/foglamp/south"
+    result = get_request(foglamp_url, get_url)
+    assert len(result["services"])
+    assert "name" in result["services"][0]
+    assert name == result["services"][0]["name"]
+
+
+def verify_ping(foglamp_url):
+    get_url = "/foglamp/ping"
+    ping_result = get_request(foglamp_url, get_url)
+    assert "dataRead" in ping_result
+    assert 0 < ping_result['dataRead'], "data NOT seen in ping header"
+    return ping_result
+
+
+def verify_in_asset(foglamp_url, ASSET_NAME):
+    ASSET_NAME = "http-" + ASSET_NAME
+    get_url = "/foglamp/asset"
+    result = get_request(foglamp_url, get_url)
+    assert len(result)
+    assert "assetCode" in result[0]
+    assert ASSET_NAME == result[0]["assetCode"]
+    return result[0]
+
+
+class TestPython35:
+    def setup_class(cls):
+        try:
+            subprocess.run(["cd $PROJECT_ROOT/tests/system/lab && ./remove"], shell=True, check=True)
+        except subprocess.CalledProcessError:
+            assert False, "remove package script failed!"
+
+        try:
+            subprocess.run(["sudo apt install -y foglamp foglamp-south-http-south foglamp-filter-expression "
+                            "foglamp-filter-python35"], shell=True, check=True)
+        except subprocess.CalledProcessError:
+            assert False, "installation of packages failed"
+
+        try:
+            subprocess.run(["cd $PROJECT_ROOT/tests/system/lab && ./reset"], shell=True, check=True)
+        except subprocess.CalledProcessError:
+            assert False, "remove package script failed!"
+
+    def test_filter_python35_with_uploaded_script(self, foglamp_url, retries, wait_time):
+        add_south_http(foglamp_url, "South_http #1")
+        time.sleep(wait_time * 2)
+        verify_south_added(foglamp_url, "South_http #1", retries)
```

Comment:
Please add constant variable `HTTP_SOUTH_SVC_NAME=South_http #1` inside class and use it further